src/content/content.js

```
@@ -65,71 +84,122 @@ injectJs(chrome.runtime.getURL("injected.js"));
+    default:
+      logDebug("Unknown request received", request.greeting);
   }
+  return true;
 });
 
 // Function to process filename
-function newFunction(filename) {
-  if (filename === "") {
+function processFilename(filename) {
+  if (!filename) {
     filename = "nofilename";
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)